### PR TITLE
fix: 同步数据库时, 忽略recycle_bin改为__recycle_bin__

### PR DESCRIPTION
**Pull request by shaozi17**
阿里云的recycle表名实际为__recycle_bin__

### CODE REVIEW

**Comment by ssongliu** [comment]
感谢贡献！这里有一个 issue 提到 recycle_bin 也得忽略掉；可以修改一下 pr，增加一个判断。

https://github.com/1Panel-dev/1Panel/issues/2589

File: backend/utils/mysql/client/local.go [hunk 1]
```diff
@@ -269,7 +269,7 @@ func (r *Local) SyncDB(version string) ([]SyncDBInfo, error) {
 		if len(parts) != 2 {
 			continue
 		}
-		if parts[0] == "SCHEMA_NAME" || parts[0] == "information_schema" || parts[0] == "mysql" || parts[0] == "performance_schema" || parts[0] == "sys" || parts[0] == "recycle_bin" {
+		if parts[0] == "SCHEMA_NAME" || parts[0] == "information_schema" || parts[0] == "mysql" || parts[0] == "performance_schema" || parts[0] == "sys" || parts[0] == "__recycle_bin__" {
 			continue
```
Comment (ssongliu) [inline backend/utils/mysql/client/local.go:273]: if parts[0] == "SCHEMA_NAME" || parts[0] == "information_schema" || parts[0] == "mysql" || parts[0] == "performance_schema" || parts[0] == "sys" || parts[0] == "recycle_bin" || parts[0] == "__recycle_bin__"

**Review by ssongliu — COMMENTED** [review]
/lgtm

**Comment by wanghe-fit2cloud** [comment]
Refs #2658

**Pull request merged**